app/components/modals/event-delete-modal.js

```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+import ModalBase from 'open-event-frontend/components/modals/modal-base';
+import FormMixin from 'open-event-frontend/mixins/form';
+import { lowerCase } from 'lodash';
+
+const { computed } = Ember;
+
+export default ModalBase.extend(FormMixin, {
+  isSmall: true,
+
+  actions: {
+    deleteEvent() {
+      this.close();
+    }
+  },
+
+  doesNameMatch: computed('confirmName', function() {
+    return lowerCase(this.get('confirmName')) !== lowerCase(this.get('eventName'));
```

Comment:
Instead of importing and using this, you could use `String.prototype.toLowerCase()` ? Similar performance.